IyiKuyoro/RMW-React-Frontend
# #160925517 Redirect users to the rides page on signin

#### What does this PR do?
Redirects a sign in user to the rides page
#### Description of Task to be completed?
> - Get the email and password inputted
> - Validate email and password
> - Send a [GET] request for user sign-in
> - If the request is successful, store token and ID and redirect to rides page
#### How should this be manually tested?
1. Clone the repository.
2. Checkout the `ft-user-should-be-directed-to-ride-page-160925517` branch.
3. Run `npm install` on your local machine.
4. Run `npm run start_dev`
5. Navigate to `localhost:8080` and sign in with an existing account
#### What are the relevant pivotal tracker stories?
#160925517
